client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -206,9 +206,11 @@ class SiteImporterInputPane extends React.Component {
 		switch ( this.props.importerData.engine ) {
 			case 'wix':
 				return (
-					<div>
+					<div className="site-importer__site-importer-url-format">
 						<p>
-							{ this.props.translate( 'Please use one of following formats for the site URL:' ) }
+							{ this.props.translate(
+								'Please use one of the following formats for the site URL:'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.